enlighten/scope/RegionControlFeature.py

```diff
@@ -49,6 +49,7 @@ def region_callback(self):
         log.debug(f"setting region to {n}")
 
         # update locally, to cause wavelengths / wavenumbers to be updated
+        # SB: needed? isn't this call generated using process_f in the next line ?
```

Comment:
Possibly no longer needed.  This was probably written when ENLIGHTEN was multi-process, and there were literally different SpectrometerSettings objects in different processes.